### ISSUE TITLE
SENet normalization of input

### ISSUE THREAD

**Issue by homelifes** [Cadene/pretrained-models.pytorch#200]
Hi. May I ask for SENet154 model in repo, the input needs to be normalized both by:
1- First dividing by 255 (having a range between 0-1)
2- Normalizing by mean and std 

Or is it only 2 without 1? I have seen some networks in your repo only use the second without the pixels being in range 0-1
